apps/router-e2e/__e2e__/compiler/app/index.tsx

```diff
@@ -1,12 +1,18 @@
 import { useEffect, useState } from 'react';
 import { Text } from 'react-native';
 
-import { useBananas } from './hooks/useFruit';
+import { useBananas, useFruit } from './hooks/useFruit';
 
 export default function Page() {
-  useBananas();
   const [isMounted, setIsMounted] = useState(false);
 
+  useBananas();
+
+  const fruits = useFruit();
+  // if (fruits !== 'Fresh Fruits are delicious!') {
+  //   throw new Error(`Expected 'Fresh Fruits are delicious!', got '${fruits}'`);
+  // }
```

Comment:
Is this comment still necessary? 😄

Reply:
Good catch, should be uncommented so the test fails, if the fruit is exported badly.

Fixed in [`f7a1980` (#38894)](https://github.com/expo/expo/pull/38894/commits/f7a1980d3d273e7eed2a937a756de9c1c9488c1e)